src/styles/global.css

```
@@ -37,6 +37,24 @@
   --breadcrumb-current: rgba(17, 16, 13, 0.38);
   --breadcrumb-sep: rgba(17, 16, 13, 0.22);
 
+  /* Heading scale (#455): display/page/article/section tiers mirror the motion
+     token pattern so page families do not hand-roll near-identical h1/h2
+     values. */
+  --font-heading: "Cormorant Garamond", Garamond, serif;
```

Stylelint (17.12.0)

[error] Expected "Garamond" to be "garamond" (value-keyword-case)

(value-keyword-case)